doc/source/how-to/vulnerabilities.rst

```diff
@@ -0,0 +1,150 @@
+
+Vulnerability disclosure
+------------------------
+
+When a vulnerability is detected and a decision is made to address it, the repository maintainers should
+create a private fork of the repository and create a pull request with the fix. Information on
+how to create such a temporary fork to resolve a vulnerability can be found in `Github's documentation`_.
+When opened, the pull request should be reviewed in depth and include tests to ensure that the
+vulnerability is fixed. Once the pull request is merged, the repository maintainers should create a new
+release with the fix and update the changelog accordingly.
+
+The release should be tagged with a new version number where the patch value has been incremented,
+and the changelog should include a note about the vulnerability and the fix. The note should include
+this information:
+
+- The CVE number of the vulnerability (if applicable)
+- A description of the vulnerability and its potential consequences
+- A description of the fix and how it addresses the vulnerability
+- A link to the pull request that fixed the vulnerability
+
+Additionally, the security advisory should be published on the repository's **Security** tab.
+This advisory should include the same information as the changelog note, in other words the
+CVE number, the date of the advisory, and the status of the advisory (such as published or withdrawn).
+
+A reference of a published security advisory can be found here: `PyAnsys Geometry subprocess advisory`_.
+This advisory was published in the PyAnsys Geometry repository and includes information about a vulnerability
+in which users could execute arbitrary code on the system by using one of this library's functions.
+
+Ensuring compliance across the PyAnsys ecosystem
+------------------------------------------------
+
+The PyAnsys Core team is responsible for ensuring that the ``ansys/actions/check-vulnerabilities`` action is up to date and that it is
+being used in all PyAnsys repositories considered as libraries (that is, Python packages shipped to PyPI).
+Repository maintainers are responsible for ensuring that the action is implemented correctly and that the
+results are reviewed regularly.
+
+.. _metapackage: https://github.com/pyansys/pyansys
+.. _CVE: https://www.cve.org/
+.. _Safety: https://pyup.io/safety/
+.. _Bandit: https://bandit.readthedocs.io/en/latest/
+.. _the action's documentation: https://actions.docs.ansys.com/version/stable/vulnerability-actions/index.html#check-vulnerabilities-action
+.. _PyACP security considerations: https://acp.docs.pyansys.com/version/dev/user_guide/security_considerations.html
+.. _Github's documentation: https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability
+.. _PyAnsys Geometry subprocess advisory: https://github.com/ansys/pyansys-geometry/security/advisories/GHSA-38jr-29fh-w9vm
```

Comment:
This links should be added to the ``links.rst`` file, unless  there is a reason not to do it.